test/functional/kernel/hyperplane/test-kernel-hyperplane-2D.cpp.in

```diff
@@ -0,0 +1,212 @@
+
+using OpenMPKernelHyperplaneExecPols =
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::omp_parallel_for_exec,
+        RAJA::statement::Hyperplane<1, RAJA::seq_exec, RAJA::ArgList<2>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >,
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::omp_parallel_for_exec,
+        RAJA::statement::Hyperplane<2, RAJA::seq_exec, RAJA::ArgList<1>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >
+  >;
+
+#endif  // RAJA_ENABLE_OPENMP
+
+#if defined(RAJA_ENABLE_TARGET_OPENMP)
+
+using OpenMPTargetKernelHyperplaneExecPols =
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::omp_target_parallel_for_exec<16>,
+        RAJA::statement::Hyperplane<1, RAJA::seq_exec, RAJA::ArgList<2>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >,
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::omp_target_parallel_for_exec<16>,
+        RAJA::statement::Hyperplane<2, RAJA::seq_exec, RAJA::ArgList<1>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >
+  >;
+
+#endif  // RAJA_ENABLE_TARGET_OPENMP
+
+#if defined(RAJA_ENABLE_TBB)
+
+using TBBKernelHyperplaneExecPols =
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::tbb_for_exec,
+        RAJA::statement::Hyperplane<1, RAJA::seq_exec, RAJA::ArgList<2>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >,
+
+    RAJA::KernelPolicy<
+      RAJA::statement::For<0, RAJA::tbb_for_exec,
+        RAJA::statement::Hyperplane<2, RAJA::seq_exec, RAJA::ArgList<1>, RAJA::seq_exec,
+          RAJA::statement::Lambda<0>
+        >
+      >
+    >
+  >;
+
+#endif  // RAJA_ENABLE_TBB
+
+#if defined(RAJA_ENABLE_CUDA)
+
+using CudaKernelHyperplaneExecPols =
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::CudaKernel<
+        RAJA::statement::For<0, RAJA::cuda_block_x_loop,
+          RAJA::statement::For<1, RAJA::cuda_thread_x_direct,
+            RAJA::statement::Hyperplane<2, RAJA::seq_exec, RAJA::ArgList<1>,
+              RAJA::statement::Lambda<0>,
+              RAJA::statement::CudaSyncThreads
+            >
+          >
+        >
+      >
+    >,
+
+    RAJA::KernelPolicy<
+      RAJA::statement::CudaKernel<
+        RAJA::statement::For<0, RAJA::cuda_block_x_loop,
+          RAJA::statement::For<1, RAJA::cuda_thread_x_loop,
+            RAJA::statement::Hyperplane<2, RAJA::seq_exec, RAJA::ArgList<1>,
+              RAJA::statement::Lambda<0>,
+              RAJA::statement::CudaSyncThreads
+            >
+          >
+        >
+      >
+    >
+
+  >;
+
+#endif  // RAJA_ENABLE_CUDA
+
+#if defined(RAJA_ENABLE_HIP)
+
+using HipKernelHyperplaneExecPols =
+  camp::list<
+
+    RAJA::KernelPolicy<
+      RAJA::statement::HipKernel<
+        RAJA::statement::For<1, RAJA::hip_thread_x_direct,
```

Comment:
The HIP exec policies have a different structure than the CUDA policies. Shouldn't they be the same?

Reply:
Good catch, I forgot the update the HIP policy.